gratipay/billing/exchanges.py

```diff
@@ -82,6 +83,7 @@ def create_card_hold(db, participant, amount):
             'options': { 'submit_for_settlement': False },
             'custom_fields': {'participant_id': participant.id}
         })
+        ref = result.transaction.id
```

Comment:
To make sure, after this PR, all the inserted records afterwards will have ref field as non-null?